qt/applications/workbench/workbench/plotting/plotscriptgenerator/__init__.py

```diff
@@ -44,7 +44,7 @@ def generate_script(fig, exclude_headers=False):
         axes.set_xlabel and axes.set_ylabel()
         axes.set_xlim() and axes.set_ylim()
         axes.set_xscale() and axes.set_yscale()
-        axes.legend().draggable()     (if legend present)
+        axes.legend().draggable(true)     (if legend present)
```

Comment:
Shouldn't this either say `axes.legned().draggable()` or `axes.legend.set_draggable(True)` since those are the two possible lines that the generated script will have?

Reply:
Update the comment